.github/workflows/prettier.yml

```diff
@@ -0,0 +1,30 @@
+name: "Prettier"
+
+on:
+  pull_request:
+    branches: [develop]
```

Comment:
I'd be tempted to allow this to run on all non-`main` branches, so it can run also on:
- direct commits in `develop`
- commits on other branches prior to creating a PR, so that one has a chance to fix things if necessary prior to a PR creation

Reply:
I think this is a good suggestion, I added issue #809.